staging/src/k8s.io/metrics/pkg/client/clientset_test/clientset_test.go

```diff
@@ -39,3 +43,148 @@ func TestFakeList(t *testing.T) {
 		t.Errorf("Unexpected error: %v", err)
 	}
 }
+
+func TestV1alpha1PodMetricsesList(t *testing.T) {
```

Comment:
Apologies for the verbose tests. I wanted to do a table test, but am not sure how to parameterize the function calls or types. Please suggest better DRYer ways so I can make my tests beautiful.